src/burn/engine/core.cpp

```diff
@@ -75,29 +77,22 @@ extern "C" HRESULT CoreInitialize(
     )
 {
     HRESULT hr = S_OK;
+    LPWSTR sczSanitizedCommandLine = NULL;
     LPWSTR sczStreamName = NULL;
     BYTE* pbBuffer = NULL;
     SIZE_T cbBuffer = 0;
     BURN_CONTAINER_CONTEXT containerContext = { };
     LPWSTR sczOriginalSource = NULL;
 
-    // parse command line
-    hr = ParseCommandLine(wzCommandLine, &pEngineState->command, &pEngineState->companionConnection, &pEngineState->embeddedConnection, &pEngineState->mode, &pEngineState->automaticUpdates, &pEngineState->fDisableSystemRestore, &sczOriginalSource, &pEngineState->elevationState, &pEngineState->fDisableUnelevate, &pEngineState->log.dwAttributes, &pEngineState->log.sczPath, &pEngineState->registration.sczActiveParent, &pEngineState->sczIgnoreDependencies, &pEngineState->registration.sczAncestors);
-    ExitOnFailure(hr, "Failed to parse command line.");
-
-    // initialize variables
+    // Initialize variables.
     hr = VariableInitialize(&pEngineState->variables);
     ExitOnFailure(hr, "Failed to initialize variables.");
 
-    // retain whether burn was initially run elevated
```

Comment:
Your merge looks good, though I did purposely delete this comment since BURN_BUNDLE_ELEVATED is not there to "retain whether burn was initally run elevated".